lib/kiev/shoryuken/request_logger.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Kiev
+  module Shoryuken
+    class RequestLogger
+      NEW_LINE = "\n"
+      ARGS = "args"
+
+      def call(_worker, _queue, _message, body)
+        began_at = Time.now
+        error = nil
+
+        begin
+          return_value = yield
+        rescue Exception => exception # rubocop:disable Lint/RescueException
```

Comment:
just wondering: is there a specific reason you have to rescue `Exception` and not `RuntimeError`? AFAIK `Exception` is reserved for process-related problems like memory insufficiency etc, not the code/runtime errors.

Reply:
It's a choice made in implemenation for Sidekiq. I don't know the details behind it.
I was following it closely in hopes to easily extract a common base class from them.

That's probably "no, no reason". And narrowing down to `RuntimeError` may be a good idea.